Tests/Build/DescribeTests.swift

```diff
@@ -25,19 +25,39 @@ final class DescribeTests: XCTestCase {
                 var clang: String { fatalError() }
             }
 
-            try POSIX.mkdtemp("spm-tests") { prefix in
-                defer { _ = try? Utility.removeFileTree(prefix) }
-                let _ = try describe(Path.join(prefix, "foo"), .debug, [], [], [], Xcc: [], Xld: [], Xswiftc: [], toolchain: InvalidToolchain())
+            let tempDir = try TemporaryDirectory(removeTreeOnDeinit: true)
+            _ = try describe(tempDir.path.appending("foo").asString, .debug, [], [], [], Xcc: [], Xld: [], Xswiftc: [], toolchain: InvalidToolchain())
                 XCTFail("This call should throw")
-            }
         } catch Build.Error.noModules {
             XCTAssert(true, "This error should be thrown")
         } catch {
             XCTFail("No other error should be thrown")
         }
     }
 
+    func testDescribingCModuleThrows() {
+        do {
+            struct InvalidToolchain: Toolchain {
```

Comment:
The difference with the one above is that `.SWIFT_EXEC` should not `fatalError()`, as `describe()` will access that member. I could change it for both, but that would impact also the test `testDescribingNoModulesThrows`.